### ISSUE TITLE
[FE - Inspect Page] Application crash on inspect page

### ISSUE THREAD

**Issue by nikitakovaliov92** [veraPDF/verapdf-webapp-gui#59]
**File:** 
[Book catalog 3 with working image.pdf](https://github.com/veraPDF/verapdf-webapp-gui/files/4710700/Book.catalog.3.with.working.image.pdf)

Steps to repoduce: 

1. Select WCAG 2.1 All profile
2. Open inspect page
3. Select page in dropdown list
4. Click to document overview

**Actual result:** App crash. "Cannot read property 'pageIndex' of undefined"
